index.html

```diff
@@ -274,65 +247,52 @@ <h3 class="about-title">Skills</h3>
-          <label>
-            <textarea class="bio" rows="3" cols="30" placeholder="Enter text here" maxlength="500" name="bio" required></textarea>
-          </label>
-        </fieldset>
-        <div class="touch-box">
-          <button class="get-in-touch" type="submit">Get in touch</button>
-        </div>
-        <div class="lower-logo">
-          <ul>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/twittertwitter.png"
-                  alt="Twitter logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/mediummedium.png"
-                  alt=" Medium logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/githubgithub.png"
-                  alt="Github logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/angellistangelist.png"
-                  alt="Angellist logo"
-              /></a>
-            </li>
-          </ul>
-        </div>
-        <p class="footer">dexterhack800@gmail.com</p>
-      </form>
-      <img class="illustration-contact" src="Images/illustration-contact-form.png" alt="illustration-contact-form">
-    </footer>
-  </body>
+  </main>
+  <footer class="contact-form">
+    <nav class="text-layout">
+      <h4 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+        touch.</h4>
+    </nav>
+    <form method="post" action="https://formspree.io/f/mbjbdrnb" class="form-data">
+      <fieldset>
+        <label><input class="label" type="text" name="first-name" placeholder="Full name" maxlength="30"
+            required /></label>
+        <label><input class="label" type="email" name="email" placeholder="Email address" required /></label>
+        <label>
+          <textarea class="bio" rows="3" cols="30" placeholder="Type in something" maxlength="500" name="bio"
+            required></textarea>
+        </label>
+        <button class="in-touch" type="submit">Get in touch</button>
+      </fieldset>
+      <div class="touch-box">
+        <button class="get-in-touch" type="submit">Get in touch</button>
+      </div>
+    </form>
+    <img class="illustration-contact" src="Images/illustration-contact-form.png" alt="illustration-contact-form">
+    <img class="illustration-desktop" src="Images/illustration-2-contact-form-desktop@2x.png"
+      alt="illustration-desktop">
```

Comment:
Kindly make sure that these images are added to your project as background. Decorations are always background images, not `img` tags. :nerd_face: :nerd_face: 

![image](https://user-images.githubusercontent.com/98363075/193182770-39d8f4f5-f811-48e6-a485-1e102d72e3f4.png)